### ISSUE TITLE
[Bug]: Outdated docker image

### ISSUE THREAD

**Issue by DenuxPlays** [sussy-code/smov#90]
### What happened?

Hey,

I noticed that the docker image has not been updated for 2 months.
The workflow itself does not run since one month.

The workflow should work maybe github actions are disabled for this repo since the repo switched to an org?

I could rebuild the workflow and create a pr but I do not think that the workflow is the problem since no actions are being run.

**Comment by ztpn** [comment]
What could we even need to update in the docker file?
Also, the workflow is fine it's just gh, after we changed to an org actions stoped working. I fixed this on providers, on the main repo I will check later

**Comment by DenuxPlays** [comment]
> What could we even need to update in the docker file? Also, the workflow is fine it's just gh, after we changed to an org actions stoped working. I fixed this on providers, on the main repo I will check later

No the Dockerfile is fine.
The image itself is 2 months old.
![image](https://github.com/user-attachments/assets/e2c6f2b2-2012-40f7-a063-d3bb4023eb13)

Would be great if you can get the workflow back up.
It's far more convenient to let watchtower update the Image then letting a cron rebuilding the image every day or so.

**Comment by DenuxPlays** [comment]
Are there any updates?

**Comment by ztpn** [comment]
Yes, one update 
`I forgot`

**Comment by DenuxPlays** [comment]
Hahaha okay 😂

**Comment by ztpn** [comment]
I don't know why the actions aren't working

**Comment by DenuxPlays** [comment]
Sad :(

I cannot see anything.
The only thing I see is that the workflows does not run at all.
I know that usually you have to allow workflow runs but I guess thats what you tried.

**Comment by ztpn** [comment]
Yes everything is configured, I will try changing smth in the workflow, I can't remember what fixed the issue for providers

**Comment by DenuxPlays** [comment]
I am not sure but just delete the old file and commit a "new" one?
Maybe this will help?

**Comment by DenuxPlays** [comment]
Any updates?

I forked the repository and activated github actions.
Just worked fine just had to replace the github PAT with the github secret that is available by default.
